snafu/upgrade_openshift_wrapper/trigger_upgrade.py

```diff
@@ -199,6 +199,7 @@ def emit_actions(self):
         self.timestamp = time.strftime("%Y-%m-%dT%H:%M:%S")
         (
             self.init_version,
+            self.desired_version,
```

Comment:
Does the log statement on line 195 output the correct desired version? Based on your changes I don't think it will and that now it needs to be updated?

Reply:
Log line 195 prints a different variable, thats set in 194. It is an output from the cluster itself.

Reply:
Ah ok. I was confused because I thought the output from the cluster itself would be inaccurate since it seemed like the upgrade hadn't been triggered yet. My bad!